### DISCUSSION TITLE
approx_percentile inaccurate

### DISCUSSION THREAD

**Discussion by SanchitBatra97** [trinodb/trino#22388] [Q&A]
What is the accuracy we can expect from approx_percentile? I have been using it to compute an approximate median and getting values 6-7% off the actual median value which seems extremely high.

**Comment by Biglucky** [comment]
same problem ,55% of the data is zero, the approx_percentile（data,[0.5]) returns a result more than zero

**Comment by bbreak** [comment]
What you describe certainly sounds like a potential bug. @SanchitBatra97 and/or @Biglucky, if you can provide details on how to reproduce the problem, please file a new issue.
